RELEASE_NOTES.md

```diff
@@ -1,17 +1,52 @@
 _This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes.html> for the official Chef release notes._
 
-# Unreleased
+# Chef Client Release Notes 13.12
 
-## Ohai Release Nots 13.12
+## Smaller Package and Install Size
+
+Both Chef packages and on disk installations have been greatly reduced in size by trimming unnecessary installation files. This has reduced our package size on macOS/Linux by ~50% and Windows by ~12%. With this change Chef 13 is now smaller than a legacy Chef 10 package.
+
```

Comment:
We trimmed unnecessary installation files, greatly reducing the package sizes for both Chef packages and on disk installations.  have been greatly reduced in size by trimming unnecessary installation files.  MacOS/Linux packages are ~50% smaller and Windows are ~12% smaller. Chef 13 is now smaller than a legacy Chef 10 package.